chapters/en/slides/module6/module6_04.md

```diff
@@ -280,7 +268,7 @@ For an environment to be reproducible, you need to add the version string manual
 
 ## Creating environment from an environment file
 
-To create a conda environment from an `environment.yaml` file, you can use the following command:
+To create a Conda environment from an `environment.yaml` file, you can use the following command:
```

Comment:
A general comment from this slide an onward is that it would be great if if was possible to reduce the amount of text in the slide to just the keywords and include a transcript for each slide that elaborates using the long text that is currently in the slide. This will keep the format consistent with the other slides we have, what do you think @armanawn ?

Reply:
Agreed and fixed.